core/lib/multivm/src/versions/vm_latest/oracles/storage.rs

```diff
@@ -59,6 +59,11 @@ pub struct StorageOracle<S: WriteStorage, H: HistoryMode> {
 
     // Storage refunds that oracle has returned in `estimate_refunds_for_write`.
     pub(crate) returned_refunds: HistoryRecorder<Vec<u32>, H>,
+
+    // Keeps track of storage keys that were written to.
```

Comment:
Definitely a nit, but I would rather emphasize that this map does not differentiate slots that were for instance written multiple times & ended up with the same value. 

```suggestion
    // Keeps track of storage keys that were ever written to.
```